gradle.properties

```diff
@@ -1,6 +1,6 @@
 # 1. Reduce Xmx after https://github.com/gradle/gradle-private/issues/4168 is resolved
 # 2. Enable html problem report for this build with -Dorg.gradle.internal.problems.report.enabled=true
-org.gradle.jvmargs=-Xmx3000m -XX:MaxMetaspaceSize=768m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -Dorg.gradle.internal.problems.report.enabled=true
+org.gradle.jvmargs=-Xmx3100m -XX:MaxMetaspaceSize=768m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -Dorg.gradle.internal.problems.report.enabled=true
```

Comment:
Not sure what required this bump, but perhaps adding kotlin to the classpath of gradlebuild.java-shared-runtime.gradle.kts did it